src/portable/espressif/esp32s2/dcd_esp32s2.c

```diff
@@ -681,6 +702,8 @@ static void _dcd_int_handler(void* arg)
     if (otg_int & USB_SESENDDET_M)
     {
       dcd_event_bus_signal(0, DCD_EVENT_UNPLUGGED, true);
+      // FIFOs will be reassigned when the endpoints are reopen
+      dcd_allocated_fifos = 1;
```

Comment:
this one is redundant, reset on bus_reset() is good enough already. Please remove this.